docker-compose.yml

```diff
@@ -25,13 +25,14 @@ x-shared_environment: &shared_environment
   DATABASE_NAME: vapor_database
   DATABASE_USERNAME: vapor_username
   DATABASE_PASSWORD: vapor_password{{/db.is_sqlite}}{{/fluent}}
-
+  
 services:
   app:
     image: {{name_kebab}}:latest
     build:
       context: .
     environment:
+      DATABASE_URL: mongodb://db:27017/vapor_database
```

Comment:
This needs to be wrapped in a check for using MongoDB otherwise it's set for all instances. We should also move it the shared environment so it's available to the migrate command etc